examples/jax/debugging/simple_spread/recurrent/decentralised/run_ippo.py

```diff
@@ -42,20 +46,24 @@
 
 
 def main(_: Any) -> None:
-    """Example running feedforward MADQN on SMAC environment."""
-
-    # WARNING (dries): This code has not been run yet. There might still
-    # be runtime errors in the code.
-
-    # Environment
-    environment_factory = functools.partial(make_environment, map_name=FLAGS.map_name)
+    """Run main script
```

Comment:
Perhaps a comment like above could be added: 
"""Example running recurrent IPPO on debugging environment."""

Reply:
Good idea :) I updated the code.